scripts/eslint/rules/argument-trivia.cjs

```diff
@@ -180,7 +179,7 @@ module.exports = createRule({
             }
 
             const getSignature = memoize(() => {
-                if (context.parserServices?.hasFullTypeInformation) {

```

Comment:
Out of curiosity, are you going to leave this boolean around in v6 so that ts-eslint using plugins don't break?

Reply:
Hmm. We weren't planning on it, but now that you bring it up, maybe we should. Good call, thanks! Filed: https://github.com/typescript-eslint/typescript-eslint/issues/7124

Reply:
Closing the loop: we've opted to continue with the removal. I've reached out to other impacted downstream repos.

Reply:
Hm, is the new code still accurate to lock this down to configs which only have type information? Skimming that issue, it's not super clear to me what method is correct especially given what I'm reading is that this is apparently not a good idea (but, it is genuinely required in our workspace as we lint files which do not have a `tsconfig` and won't work until ProjectService is a thing).